### ISSUE TITLE
`no-unknown-property`: breaking change between versions 7.31.1 and 7.31.5

### ISSUE THREAD

**Issue by mrtnzlml** [jsx-eslint/eslint-plugin-react#3391]
Hello! 👋 I received a PR with a patch upgrade (see: https://github.com/adeira/universe/pull/4976), containing 571 new errors. It turns out there was some breaking change in the `react/no-unknown-property` rule, and it currently fails on FBT nodes, for example:

```jsx
<fbt desc="test tooltip content title" doNotExtract={true}>
  This is a test tooltip content.
</fbt>
```

Resulting in:

```text
22:14  error  Unknown property 'desc' found          react/no-unknown-property
22:48  error  Unknown property 'doNotExtract' found  react/no-unknown-property
```

Recently, I reported several issues related to FBT, and they were all fixed, and some rules support it (https://github.com/jsx-eslint/eslint-plugin-react/search?q=fbt&type=code). But I agree that FBT is a bit awkward React citizen, so not sure if this is going to be taken very seriously. 😅 

Thanks for having a look! 😎

**Comment by ljharb** [comment]
It’s a bug, not a breaking change. Thanks for the report, we’ll get it fixed!
